scripts/export-package.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+name="$1"
+type="$2"
+
+IFS=';' read -ra package_components <<< "${name// - /;}"
+
+package_prefix="${package_components[0]}"
+package_code="${package_components[1]}"
+description="${package_components[2]}"
+
+if [[ "${Custom_description:-}" ]]; then
+  description="${Custom_description}"
+fi
+
+if [[ "${Instance_url:-}" ]]; then
+  instance="$Instance_url"
+else
+  if [[ "$type" == "TRK" || "$type" == "EVT" ]]; then
+    case "$DHIS2_version" in
+      "2.36")
+        instance="https://metadata.dev.dhis2.org/tracker_dev"
```

Comment:
I would maybe extract host urls into vars declared at the top of the file for simplicity

Reply:
There is a typo... For 2.37 -> instance="https://who-dev.dhis2.org/tracker_dev236" (should be 237)
The logic could also be simplified:
Have a variable current dev version = 2.36. Current version can be inferred also making an API call to system/info
If version selected by user > current_dev_version
  instance="https://who-dev.dhis2.org/tracker_dev" + DHIS_VERSION.replace('.','')
  
Also these URLs will be removed at some point (when we have docker containers) so no need to put too much effort on that either

Reply:
@vilkg Done :+1:  
@stratosilva  I've updated the instance urls to match the expected case strings. Won't go into adding extra logic of inferring the versions for now, as we're close to using the Instance Manager for creating instances here.